flair/trainers/text_classification_trainer.py

```diff
@@ -233,8 +229,7 @@ def evaluate(self, sentences: List[Sentence], eval_class_metrics: bool = False,
             batches = [sentences[x:x + mini_batch_size] for x in
                        range(0, len(sentences), mini_batch_size)]
 
-            y_pred = []
-            y_true = []
+            metric = Metric('')
```

Comment:
A metric should get a meaningful name. I would suggest either `MIRCO_AVG` or the data set type (e.g. `TEST`, `DEV` or `TRAIN`).